### PR TITLE
doc 3.00

### PR DESCRIPTION
**Pull request by gerhardol**
gitextension/gitextensions#5693

draft for doc update

Replacing screenshots
Remove 2.x functionality descriptions
Remove some irrelevant desriptions
Brief add new functionality
Moved to #90: Minor refresh to coloring, matching updated dashboard

This may be split in several PRs, the documentation has not been updated much since it was created, so there are 5 years of changes.
Contributions are welcome

### CODE REVIEW

File: source/plugins.rst [hunk 1]
```diff
@@ -41,7 +43,7 @@ This list is incomplete.
 
     .. setting:: Fetch all submodules
 
-      If checked, also perform ``git fetch �all`` recursively on all configured
+      If checked, also perform ``git fetch -all`` recursively on all configured
```
Comment (drewnoakes) [inline source/plugins.rst]:
```suggestion
     If checked, also perform ``git fetch --all`` recursively on all configured
```

**Comment by gerhardol** [comment]
Progressing slowly, more than half should be done by now at least.
I am touching-up and adding information is in some places, this is something someone need to return to, the documentation has more or less not been updated in 5 years.

When I feel that I am done (or get stuck), I suggest merging directly and ask users to review and enhance. (Open a new issue for visibility and ask for help in gitter.)

**Comment by RussKie** [comment]
Agree and support.
Thank you!

File: source/plugins.rst [hunk 2]
```diff
@@ -30,6 +101,96 @@ You should have GitFlow installed to use this plugin.
...
+
+    .. setting:: Refresh view after fetch
+
+      If checked, the commit log and branch labels will be refreshed after the fetch.
+      If you are browsing the commit log and comparing revisions you may wish
+      to disable the refresh to avoid unexpected changes to the commit log.
+
+    .. setting:: Fetch all submodules
+
+      If checked, also perform ``git fetch -all`` recursively on all configured
```
Comment (drewnoakes) [inline source/plugins.rst]:
```suggestion
      If checked, also perform ``git fetch --all`` recursively on all configured
```

**Comment by gerhardol** [comment]
I have refreshed the documentation, replaced all screenshots, added a few missing pieces of info and touched up a few incorrect pieces. There is still a lot of missing information (not much updates since @martinqt wrote this about five years ago) and I have very likely introduced new errors.

However, I believe that the total is much improved suggest to merge this as is without much review and point the documentation to point to this. I will then open a new PR in the main repo with a request to review the documentation by users. This is much easier for them if a "build" already exists and they have something to base improvements on. I have noticed some issues in GE too when trying to get the screenshots.

I will squash this tomorrow (even if approvals was some time ago, I believe this is OK and the best way forward).

@RussKie @spdr870
To complete the documentation someone with admin privileges need to create the release/3.00 branch and point that to master, then move 'latest' branch too (that will retrigger a a build on readthedocs).

**Pull request merged**